File: scratch_inspect_errors.py
import re
from bs4 import BeautifulSoup

with open("c:/Users/LENOVO/vedata-test/page_source_employee_error.html", "r", encoding="utf-8") as f:
    html = f.read()

soup = BeautifulSoup(html, "html.parser")

print("=== ALL INPUTS & VALUES AFTER ERROR ===")
for inp in soup.find_all(["input", "textarea", "select"]):
    inp_id = inp.get('id')
    inp_type = inp.get('type')
    inp_name = inp.get('name')
    inp_val = inp.get('value', '')
    
    # Try to find label
    label_text = ""
    if inp_id:
        lbl = soup.find("label", attrs={"for": inp_id})
        if lbl:
            label_text = lbl.get_text(strip=True)
            
    if not label_text:
        parent = inp.parent
        while parent and parent.name != "body":
            if "v-field" in parent.get("class", []):
                lbl = parent.find("label")
                if lbl:
                    label_text = lbl.get_text(strip=True)
                break
            parent = parent.parent
            
    print(f"Tag={inp.name} | id={inp_id} | type={inp_type} | label='{label_text}' | val='{inp_val}'")

File: scratch_inspect_errors.js
const fs = require('fs');

try {
  const html = fs.readFileSync('c:/Users/LENOVO/vedata-test/page_source_employee_error.html', 'utf8');
  
  // Find all <input ...> tags
  const inputRegex = /<(input|textarea|select)\b([^>]*)\/?>/gi;
  let match;
  console.log("=== ALL INPUTS & VALUES IN ERROR PAGE ===");
  while ((match = inputRegex.exec(html)) !== null) {
    const tag = match[1];
    const attrsStr = match[2];
    
    // Parse attributes
    const idMatch = attrsStr.match(/id=["']([^"']*)["']/i);
    const typeMatch = attrsStr.match(/type=["']([^"']*)["']/i);
    const valueMatch = attrsStr.match(/value=["']([^"']*)["']/i);
    const labelMatch = attrsStr.match(/aria-label=["']([^"']*)["']/i);
    
    const id = idMatch ? idMatch[1] : '';
    const type = typeMatch ? typeMatch[1] : '';
    const value = valueMatch ? valueMatch[1] : '';
    const label = labelMatch ? labelMatch[1] : '';
    
    // Try to find if there is a value inside select or textarea
    let innerValue = '';
    if (tag === 'textarea' || tag === 'select') {
      const closeTagRegex = new RegExp(`</${tag}>`, 'i');
      const startIdx = match.index + match[0].length;
      const endMatch = closeTagRegex.exec(html.substring(startIdx));
      if (endMatch) {
        innerValue = html.substring(startIdx, startIdx + endMatch.index).trim();
        innerValue = innerValue.replace(/<[^>]*>/g, '').trim(); // Remove nested tags
      }
    }
    
    console.log(`Tag=${tag.toUpperCase()} | id=${id} | type=${type} | label='${label}' | val='${value || innerValue}'`);
  }
} catch (e) {
  console.error("Error reading file:", e);
}
